tests/lifecycle/postgresql_cases_test.go

```diff
@@ -190,3 +200,75 @@ func testPostgreSQLCreds(credentials map[string]interface{}) error {
 	}
 	return nil
 }
+
+// nolint: lll
+func createVirtualNetworkForPostgres(
+	ctx context.Context,
+	resourceGroup string,
+	parent *service.Instance,
+	pp *map[string]interface{},
+) error {
+	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
+	azureConfig, err := getAzureConfig()
+	if err != nil {
+		return fmt.Errorf("error getting azure config %s", err)
+	}
+	authorizer, err := getBearerTokenAuthorizer(azureConfig)
+	if err != nil {
+		return fmt.Errorf("error getting authorizer %s", err)
+	}
+	virtualNetworksClient := networkSDK.NewVirtualNetworksClientWithBaseURI(
+		azureConfig.Environment.ResourceManagerEndpoint,
+		azureConfig.SubscriptionID,
+	)
+	virtualNetworksClient.Authorizer = authorizer
+	virtualNetworkName := uuid.NewV4().String()
+	subnetName := "default"
+	vnResult, err := virtualNetworksClient.CreateOrUpdate(
+		ctx,
+		resourceGroup,
+		virtualNetworkName,
+		networkSDK.VirtualNetwork{
+			Location: to.StringPtr("eastus"),
+			VirtualNetworkPropertiesFormat: &networkSDK.VirtualNetworkPropertiesFormat{
+				AddressSpace: &networkSDK.AddressSpace{
+					AddressPrefixes: &[]string{"172.19.0.0/16"},
+				},
+				Subnets: &[]networkSDK.Subnet{
+					{
+						Name: &subnetName,
+						SubnetPropertiesFormat: &networkSDK.SubnetPropertiesFormat{
+							AddressPrefix: to.StringPtr("172.19.0.0/24"),
+						},
+					},
+				},
+			},
+		},
+	)
+	if err != nil {
+		return fmt.Errorf("error creating virtual network %s", err)
+	}
+	if err = vnResult.WaitForCompletion(
+		ctx,
+		virtualNetworksClient.Client,
+	); err != nil {
+		return fmt.Errorf("error waiting for virtual network creation complete %s", err)
+	}
+	subnetID := fmt.Sprintf(
+		"/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Network/virtualNetworks/%s/subnets/%s",
+		azureConfig.SubscriptionID,
+		resourceGroup,
+		virtualNetworkName,
+		subnetName,
+	)
+
+	(*pp)["virtualNetworkRules"]: []interface{}{
```

Comment:
First time writing any Go, and I'm having a syntax problem here I think.
I'm getting 
`illegal label declaration`
Full error:
```
docker run --rm -e CGO_ENABLED=0 -e AZURE_SUBSCRIPTION_ID=${AZURE_SUBSCRIPTION_ID} -e AZURE_TENANT_ID=${AZURE_TENANT_ID} -e AZURE_CLIENT_ID=${AZURE_CLIENT_ID} -e AZURE_CLIENT_SECRET=${AZURE_CLIENT_SECRET} -e TEST_MODULES=${TEST_MODULES} -v $(pwd):/go/src/github.com/Azure/open-service-broker-azure -w /go/src/github.com/Azure/open-service-broker-azure quay.io/deis/lightweight-docker-go:v0.2.0 go test -parallel 12 -timeout 80m github.com/Azure/open-service-broker-azure/tests/lifecycle -v
# github.com/Azure/open-service-broker-azure/tests/lifecycle
tests/lifecycle/postgresql_cases_test.go:266:30: illegal label declaration
FAIL	github.com/Azure/open-service-broker-azure/tests/lifecycle [setup failed]
make: *** [test-service-lifecycles] Error 1
```

Any hints?
I assume it's because its an array, I tried to base it on the firewalls rule definition